Tests/FilesTests/FilesTests.swift

```diff
@@ -260,6 +260,19 @@ class FilesTests: XCTestCase {
         }
     }
     
+    func testCopyingFiles() {
+        performTest {
+            let file = try folder.createFile(named: "A")
+            
+            let subfolder = try folder.createSubfolder(named: "folder")
+            try file.copy(to: subfolder)
+            try XCTAssertNotNil(folder.file(named: "A"))
+            try XCTAssertNotNil(subfolder.file(named: "A"))
+            try XCTAssertEqual(file.read(), subfolder.file(named: "A").read())
```

Comment:
Probably a good idea to give the file some content, so that you're not just comparing empty `Data` structs.

Reply:
I thought about it. I agree, will add some string content